oauth_client_integration_test.go

```diff
@@ -429,6 +429,30 @@ func TestOAuthClientsUpdate_rsaKeyPair(t *testing.T) {
 		assert.Equal(t, newKey, oc.Key)
 	})
 
+	t.Run("updates organization scoped", func(t *testing.T) {
```

Comment:
I don't know if it makes sense to mix this in with the RSA key pair tests -- maybe consider adding a generic `TestOAuthClientsUpdate` function?